tsl/test/sql/continuous_aggs_union_view.sql

```diff
@@ -0,0 +1,119 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+-- disable background workers to make results reproducible
+\c :TEST_DBNAME :ROLE_SUPERUSER
+SELECT _timescaledb_internal.stop_background_workers();
```

Comment:
FWIW, wondering if test shouldn't run without background workers  OFF by default? Should probably explicitly enable for the tests that require them.

Reply:
sounds like a good idea and hopefully reduces flaky tests